src/com/untamedears/realisticbiomes/listener/PlayerListener.java

```diff
@@ -45,10 +49,19 @@ public void onPlayerInteractEvent(PlayerInteractEvent event) {
 			return;
 		}
 		
-		Plant plant = null;
-		
 		Block block = event.getClickedBlock();
 		
+		if (((List<ItemStack>)block.getDrops()).get(0).getType() == event.getMaterial()) {

```

Comment:
Surely this cast is unsafe and could be changed to something like `block.getDrops().iterator().next()`?

Reply:
According to the bukkit docs, it is a safe cast. And don't call me Shirley.